WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -4,41 +4,18 @@ final class BlogDashboardViewController: UIViewController {
 
     var blog: Blog?
 
-    enum Section: CaseIterable {
-        case quickLinks
-        case posts
-    }
-
-    // FIXME: temporary placeholder
-    private let quickLinks = ["Quick Links"]
-    private let posts = ["Posts"]
+    private var viewModel: BlogDashboardViewModel!
```

Comment:
Something we may want to address in a future PR:

Could we make this a non-optional `BlogDashboardViewModel`? And maybe we can lazy init the viewModel?

I think we may want to inject a non-optional `Blog` in the init for this VC.

Reply:
Fixed in 2898f813658cac9559bcd35558e16b7ae41b744e